torch/csrc/jit/tracer.cpp

```diff
@@ -102,6 +102,13 @@ Value* getValueTrace(const IValue& var) {
     }
 
     // Didn't find it. Bake in a constant
+    if (ten.requires_grad()) {
+      std::ostringstream oss;
+      oss << "Cannot insert a Tensor that requires grad as a constant:\n";
```

Comment:
We should probably have a reference to a specific location in the code, as well as possible remediations (trace the module, pass the parameter in explicitly).

Reply:
Do we have access to where in the code we're at?